src/main/java/seedu/us/among/model/endpoint/Method.java

```diff
@@ -0,0 +1,85 @@
+package seedu.us.among.model.endpoint;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.us.among.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Endpoint's Method in the address book. Guarantees: immutable; is
+ * valid as declared in {@link #isValidMethod(String)}
+ */
+public class Method {
```

Comment:
It will be good to include a check that method can only be "POST" or "GET" or "PUT" or "PATCH" or "DELETE", just like how the email checked that it had an email had @ and .com

Reply:
Else, the method will throw an error

Reply:
I think the check has been performed in `isValidMethod` below :D

Reply:
oh sorry :<